mail_queue_job/README.rst

```diff
@@ -0,0 +1,92 @@
+====================
+Mail Connector Queue
```

Comment:
`Mail Queue Job`?

Reply:
This file should be re-generated automatically after the merge?
And the name is automatically set by the name of the directory I suppose.

Reply:
I was not sure from where the name comes from, it's from the manifest file: https://github.com/OCA/maintainer-tools/blob/80f894b594810200386f176e19cd79bf4d7be8c5/tools/gen_addon_readme.template#L11

So indeed this is going to be correct after the automatic update as the name in the manifest is good.